### PR TITLE
Added example of beta series regression

### PR DESCRIPTION
**Pull request by bthirion**

### CODE REVIEW

**Comment by kchawla-pi** [comment]
I have made some changes. If you are satisfied we can merge.

**Comment by bthirion** [comment]
LGTM. Thx for the improvements.
Best,
Bertrand

**Comment by kchawla-pi** [comment]
Add an entry in Whats_new under NEW

File: doc/whats_new.rst [hunk 1]
```diff
@@ -11,7 +11,7 @@ New
 
 * Use :func:`nistats.reporting.make_glm_report` to easily generate HTML reports from fitted first and second level models and contrasts.
 * New dataset fetcher, :func:`nistats.datasets.fetch_language_localizer_demo_dataset` , BIDS 1.2 compatible.
-* New example showcasing the use of a GLM to get beta maps for decoding experiments (aka beta-regression) in :func:`examples/05_complete_examples/plot_haxby_block_classification.py`
+* New example showcasing the use of a GLM to get beta maps for decoding experiments (aka beta-regression).
```
Comment (kchawla-pi) [inline doc/whats_new.rst]: We should add the link. I'll find another way.

Comment (bthirion) [reply doc/whats_new.rst]: Glad to learn how you do that !

**Pull request merged**